spec/facebook_ads/configuration_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe FacebookAds::Configuration do
+  subject { described_class.new }
+
+  describe '#valid?' do
+    it 'is false by default' do
+      expect(subject.valid?).to be_falsey
+    end
+
+    it 'is true after setting an access_token' do
+      subject.access_token = '123'
+      expect(subject.valid?).to be_truthy
```

Comment:
Any method that ends in a ? should return true or false and nothing else. The expectation should reflect that and check for true instead of truthy.